runtime/metrics.rs

```diff
@@ -1,6 +1,36 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+use serde::Serialize;
+
 #[derive(Default, Debug)]
-pub struct Metrics {
+pub struct RuntimeMetrics {
+  pub ops: HashMap<String, OpMetrics>,
```

Comment:
If you change the type to `HashMap<&'static str, OpMetrics>`, you don't have to to_string() + clone() everywhere.

(Requires a change to `runtime/ops/plugin.rs` but you could `Box::leak()` it if you don't want to break API.)

Reply:
> If you change the type to HashMap<&'static str, OpMetrics>, you don't have to to_string() + clone() everywhere.

Yeah. This was my initial implementation, but then I got stuck on `runtime/ops/plugin.rs`. Using `Box::leak` seems to work. Didn't know about that before. Thanks :-)